modules/services/fb_streetview_ai_suggestions.js

```diff
@@ -0,0 +1,499 @@
+import _forEach from 'lodash-es/forEach';
+
+import { dispatch as d3_dispatch } from 'd3-dispatch';
+import { xml as d3_xml } from 'd3-fetch';
+
+import { coreGraph, coreTree } from '../core';
+import { osmEntity, osmNode, osmWay, osmRelation } from '../osm';
+import { utilRebind, utilStringQs, utilTiler } from '../util';
+
+// constants
+var APIROOT = 'https://www.gaoxm.sb.facebook.com/maps/ml_roads';
+var TILEZOOM = 16;
+var tiler = utilTiler().zoomExtent([TILEZOOM, TILEZOOM]);
+var dispatch = d3_dispatch('loadedData');
+
+var _datasets = {};
+var _deferredAiFeaturesParsing = new Set();
+var _off;
+
+
```

Comment:
I know this is just for demo purposes, but I'd like to make sure we extract out any common service code into a helper file- a lot of these methods are identical to the ones in modules/services/fb_ai_features.js!

Reply:
to be clear, we can refactor later when we have a better idea of what the service might look like. Just a 'todo' really.